docs/submission.md

```diff
@@ -87,3 +87,17 @@ We need a **Command Line Interface (CLI)** to execute your code. This is the com
 * For MATLAB, this is defined by the MATLAB Compiler
 * For Python, several options are well-described here: https://codeburst.io/building-beautiful-command-line-interfaces-with-python-26c7e1bb54df 
 
+## Showing Progress
+As a bonus feature, you may let the user know the progress of the code execution, if you are able to extract it during runtime. Examples of applicable cases include:
+* Knowledge of simulation time (e.g. the code simulates a process for 10 seconds, therefore knowledge of the second of simulation time gives #seconds*10%)
```

Comment:
FYI: do you know that you can use latex formulas here? see https://upupming.site/docsify-katex/docs/#/supported

```markdown
$seconds * 10 %$
```

Reply:
thanks! makes sense